src/stores/AutoRageshakeStore.ts

```diff
@@ -85,20 +85,26 @@ export default class AutoRageshakeStore extends AsyncStoreWithClient<IState> {
             if (now - this.state.lastRageshakeTime < RAGESHAKE_INTERVAL) { return; }
 
             await this.updateState({ lastRageshakeTime: now });
-            const matchingIssue = await sendBugReport(SdkConfig.get().bug_report_endpoint_url, {
-                userText: "Auto-reporting decryption error (recipient)",
-                sendLogs: true,
-                label: "Z-UISI",
-            });
 
-            const messageContent = {
+            const eventInfo = {
                 "event_id": ev.getId(),
                 "room_id": ev.getRoomId(),
                 "session_id": sessionId,
                 "device_id": wireContent.device_id,
                 "user_id": ev.getSender(),
                 "sender_key": wireContent.sender_key,
-                "matching_issue": matchingIssue,
+            };
+
+            const rageshakeURL = await sendBugReport(SdkConfig.get().bug_report_endpoint_url, {
+                userText: "Auto-reporting decryption error (recipient)",
+                sendLogs: true,
+                label: "Z-UISI",
+                customFields: { "auto-uisi": JSON.stringify(eventInfo) },
```

Comment:
stay consistent with one casing type between your custom fields please

Reply:
See above. @BillCarsonFr, want to make a followup PR to element-android so we can get the casing consistent?

Reply:
> See above. @BillCarsonFr, want to make a followup PR to element-android so we can get the casing consistent?

sure